### ISSUE TITLE
Dropwizard 3 support

### ISSUE THREAD

**Issue by staffors** [pac4j/dropwizard-pac4j#404]
There doesn't seem to be any Dropwizard 3 support right now in dropwizard-pac4j, but the changes required to support 3 look to be pretty simple. If I made a PR would there be any chance of getting it merged? There might be a good reason that supporting Dropwizard 3 is difficult that I'm not seeing. The hardest part might be figuring out a version number that would make sense.... 

FYI, the reason I need support for Dropwizard 3 is that my app has made extensive use of Hibernate Criteria queries and those block us from upgrading straight to 4. We'll eventually get rid of them all, but in the meantime we're planning to run Dropwizard 3. we're currently just modifying the source locally to make dropwizard-pac4j 6.0.0 work with Dropwizard 3, but it would be nicer to be able to import a compatible version of the library.

**Comment by leleuj** [comment]
Sure. Please submit a PR.

**Comment by mustafashabib** [comment]
Curious if you got around to this @staffors

**Comment by staffors** [comment]
Nope, I've been on vacation and just got back. Not sure if I'll get to it tomorrow or maybe next week?

**Comment by mustafashabib** [comment]
Cool - will keep my eyes peeled as we're in the same boat.

**Comment by staffors** [comment]
I just opened a PR for this issue (https://github.com/pac4j/dropwizard-pac4j/pull/406)

**Comment by mustafashabib** [comment]
Awesome hope it gets merged and released to maven central soon.

**Comment by leleuj** [comment]
I just cut the v5.3.0 release, it will show up in the Maven central repo within a few minutes/hours.
